src/mon/Monitor.cc

```diff
@@ -3832,8 +3832,16 @@ void Monitor::handle_command(MonOpRequestRef op)
     for (auto& n : ids) {
       if (monmap->contains(n)) {
 	wouldbe.erase(n);
+      } else {
+	  iinvalid_ids.push_back(n);
```

Comment:
Typo here.

Reply:
@NitzanMordhai https://jenkins.ceph.com/job/ceph-pull-requests/89607/consoleFull#17331088632a811ea2-3e7b-466b-84b4-d13df7e35809 is related, needs to be fixed

Reply:
Done